dart/test/benchmark_test.dart

```diff
@@ -2,6 +2,9 @@ import 'package:open_location_code/open_location_code.dart' as olc;
 import 'package:test/test.dart';
```

Comment:
_This is an automated bot comment from the TravisCI tests_<br>**File has `dartfmt` errors that must be fixed**. Here is a diff, or run `format_check.sh`:<br><pre>6c6
< var bar = (47/5).toInt();
---
> var bar = (47 / 5).toInt();</pre>